### ISSUE TITLE
parser: SyntaxError: cannot delete starred

### ISSUE THREAD

**Issue by pmp-p** [pybpc/f2format#16]
del with multiple values of trailing comma is not parsed correctly

```
echo 'a=1;del a,' | f2format -s
Traceback (most recent call last):
  File "/Users/user/bin/f2format", line 8, in <module>
    sys.exit(main())
  File "<string>", line None
bpc_utils.parsing.BPCSyntaxError: source file '<unknown>' contains the following syntax errors:
[L1C4] SyntaxError: cannot delete starred
```

**Comment by gousaiyang** [comment]
Hello! Thanks for the report. After some quick investigation, this is a bug in our dependency library [`parso`](https://github.com/davidhalter/parso). From [its source code](https://github.com/davidhalter/parso/blob/ef90bba3b3d4310c345885a1368dbfc8937765a3/parso/python/errors.py#L742) I can see that the detection logic do have false positives. It should only detect deletion of starred expressions instead of all `exprlist` nodes. Currently code like `del a,` and `del a, b` are incorrectly considered as invalid code. You can file a bug report to `parso`.

**Comment by gousaiyang** [comment]
Fixed in davidhalter/parso#168
